AEPLifecycle/Sources/LifecycleV2/LifecycleV2MetricsBuilder.swift

```diff
@@ -165,25 +166,25 @@ class LifecycleV2MetricsBuilder {
 
 }
 
-extension SystemInfoService {
+extension String {
 
-    /// Return a string for the given 'locale' identifier.
+    /// Returns a BCP formatted locale from the calling locale String.
     /// Uses the format "`Locale.languageCode`-`Locale.regionCode`".
```

Comment:
Can you update this comment to clarify?

> Uses  `Locale.identifier(.bcp47)` for iOS 16+, otherwise uses format "`Locale.languageCode`-`Locale.regionCode`"